index.html

```diff
@@ -18,6 +18,9 @@
 
   <body>
     <div id="app"></div>
+    <!-- SDK-1: Include Bundled SDK scripts -->
```

Comment:
remove this comment

Reply:
Resolved at https://github.com/arcana-network/sdk-demo/pull/5/commits/58cfeb28f8e0ede2e9f9223a2410b107ef9f5a08